core-bundle/src/Resources/contao/library/Contao/Input.php

```diff
@@ -985,14 +985,7 @@ public static function findPost($strKey)
 			return $_POST[$strKey];
 		}
 
-		$request = System::getContainer()->get('request_stack')->getMasterRequest();
-
-		// Return if the session has not been started before
-		if ($request === null || !$request->hasPreviousSession())
-		{
-			return null;
-		}
-
+		// $_SESSION access is governed by LazySessionAccess (#3971)
```

Comment:
```suggestion
		// Do not check for $request->hasPreviousSession() and early return here (see #3971)
```

As suggested by @Toflar, so we find this issue again.

Reply:
I already referenced the PR in my comment, but ok ;). Also this still applies to any `$_SESSION` access, not just this particular one in `Input::findPost`.